src/cpp/session/modules/SessionConnectionsInstaller.R

```diff
@@ -445,7 +445,8 @@
 })
 
 .rs.addFunction("odbcBundleDriverIniPath", function(name, driverPath) {
-   dir(driverPath, pattern = paste(tolower(name), ".*\\.ini", sep = ""), recursive = TRUE, full.names = T)
+.rs.addFunction("odbcBundleDriverIniPath", function(driverPath) {
```

Comment:
I changed the function signature since `name` is no longer used.

If we want to keep the same function signature, that is completely fine and I'll just revert to the one-line change.